packages/ui/SurveysList/index.tsx

```diff
@@ -20,6 +21,14 @@ interface SurveysListProps {
   surveysPerPage: number;
 }
 
+export const initialFilters: TSurveyFilters = {
+  name: "",
+  createdBy: [],
+  status: [],
+  type: [],
+  sortBy: "updatedAt",
+};
+
 export default function SurveysList({
```

Comment:
Not related to this PR, but can we remove the default export here and convert this to an arrow function (same for all other components in `packages/ui/SurveyList`) 😊